redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/redmine-projects/listProjects.xml

```diff
@@ -17,20 +17,45 @@
 ~ under the License.
 -->
 <template name="listProjects" xmlns="http://ws.apache.org/ns/synapse">
+
+    <!--Optional parameters -->
+    <parameter name="redminelimit" description="number of projects to return" />
+    <parameter name="redmineoffset" description="index of the first project to return" />
+
```

Comment:
There is another optional parameter called 'include'. So modify this method to support that too.